modular_skyrat/modules/customization/modules/language/buzzwords.dm

```diff
@@ -0,0 +1,2 @@
+/datum/language/buzzwords
+	secret = TRUE
```

Comment:
this is a language used by an actively selectable species(flypeople), this needs to be selectable